Education.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MarsQA_1.Helpers;
+using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace MarsQA_1.Pages
+{
+    class Education
+    {
+        private static IWebElement Education_ => Driver.driver.FindElement(By.XPath("//A[text()='Education']"));
+
+        public static string title_name;

```

Comment:
Generally, it's a good practice not to use statics unless it's going to be reused
https://www.infoworld.com/article/3045882/static-classes-and-static-class-members-in-c-explained.html#:~:text=The%20static%20keyword%20in%20the,static%20classes%20and%20static%20members.&text=The%20difference%20between%20a%20static,class%20are%20static%20in%20nature

Reply:
Yeah sure. As I said before I just re-used the Sign in class  given in the existing framework